binary/test/Test/Pos/Binary/BiSerialize.hs

```diff
@@ -137,16 +220,4 @@ genTestSimple sz
         , TsChar2Permuted <$> Gen.unicode <*> Gen.unicode
         ]
     bignum = 2 ^ (80 :: Integer)
-
-
-prop_TestSimple_TestIndexed_equivalent :: Property
-prop_TestSimple_TestIndexed_equivalent = H.withTests 2000 $ H.property $
-    H.forAll (Gen.sized genTestSimple) >>= \ts ->
-        B16.encode (serialize' ts)
-        ===
-        B16.encode (serialize' (simpleToIndexed ts))
-
-
-tests :: IO Bool
-tests =
-  H.checkParallel $$discover
+-}
```

Comment:
We could delete this block comment. I think it gives context for anyone wondering about the transition from `deriveSimpleBi` to deriveIndexedBi` in the future. But it could also be left in git history.

Reply:
Given what we're testing here and how the tests were generated, I like the idea of leaving this in (it'd be especially helpful if I wanted to write another golden test for some reason).